ChaosMod/Effects/db/Player/PlayerJumpJump.cpp

```diff
@@ -0,0 +1,18 @@
+/*
+	Effect by Last0xygen
+*/
+
+#include <stdafx.h>
+static int lastCheck = 0;
+
+static void OnTick()
+{
+	Ped player = PLAYER_PED_ID();
+	if (lastCheck + 500 < GET_GAME_TIMER() &&  !IS_PED_JUMPING(player))
+	{
+		lastCheck = GET_GAME_TIMER();
+		TASK_JUMP(player, false, false, false);
```

Comment:
This could also be `TASK_CLIMB` which does the same thing except it makes the player climb things if possible.